resources/views/category/show.blade.php

```diff
@@ -2,7 +2,7 @@
 
 @section('content')
 <main class="row container mx-auto">
-    <aside class="col-auto" style="min-height: calc(100vh - 160px); background-color: rgba(247, 160, 114, 0.2);">
+    <aside class="col-auto" style="min-height: calc(100vh - 160px); background-color: #FFE4D6;">
```

Comment:
If you are going to use this style repetitive times, better to use class in .css file